fullstory.go

```diff
@@ -134,5 +134,9 @@ func (c *Client) ExportData(id int) (ExportData, error) {
 		return nil, err
 	}
 
+	for _, mr := range modifyReq {
+		mr(req)
+	}
+
```

Comment:
this already exists in master, so maybe rebasing will make this go away from this PR?